diff -rupN terminatorX-3.84.old/src/tX_midiin.cc terminatorX-3.84/src/tX_midiin.cc
--- terminatorX-3.84.old/src/tX_midiin.cc	2011-11-13 07:53:51.000000000 -0500
+++ terminatorX-3.84/src/tX_midiin.cc	2013-03-24 15:59:01.904683280 -0400
@@ -447,7 +447,7 @@ gboolean tX_midiin::midi_learn_destroy(G
 
 void tX_midiin::store_connections(FILE *rc, char *indent) 
 {
-	gzFile *rz=NULL;
+	gzFile rz=NULL;
 	
 	tX_store("%s<midi_connections>\n", indent);
 
